includes/admin/class-filter-posts.php

```diff
@@ -34,6 +34,17 @@ public static function add_brands_dropdown( $post_type ) {
 			return;
 		}
 
+		$num_posts_w_brand = wp_count_terms(
```

Comment:
That's a bit of a weird variable name. I know it's about "with brand", but I think it'd be better to spell it out.

Reply:
Agreed :) I spelled it out